other/field-properties.md

```diff
@@ -17,6 +17,8 @@ Each [Schema Field Type](/tutorials/advanced-development/schema-guide.md#guide-t
 |def | depends on field type | | The default value for the field | most fields, excluding areas, singletons, objects, and arrays | |
 |searchable | `boolean` | true | If false a field will not inform sitewide search | [string](field-types/string.md), [area](field-types/area.md), [singleton](field-types/singleton.md) | |
 |limit | `int` | | sets the max number of tags in a tag field, the max number of widgets in an area, and the max number of entries in an array  |[tags](field-types/tags.md), [area](field-types/area.md), [array](field-types/array.md)  | |
-|options | `object` | | Passed on to apos.area or apos.singleton as appropriate, specifying options for the allowable widgets and so on | [area](field-types/area.md), [singleton](field-types/singleton.md) | `widgets` |
+|[options](properties/options.md) | `object` | | Passed on to apos.area or apos.singleton as appropriate, specifying options for the allowable widgets and so on | [area](field-types/area.md), [singleton](field-types/singleton.md) | `widgets` |
 |[sortify](properties/sortify.md) | `boolean` | false | If true, make sort() operations on the field case-insensitive and more intuitive  | [string](field-types/string.md), [email](field-types/email.md) | |
 |[choices](properties/choices.md) | `array` | | An array of values that the user can select from with each being an object with value and label properties | [boolean](field-types/boolean.md) [select](field-types/select.md), [checkboxes](field-types/checkboxes.md) | `label`, `value`, `showFields` |
+|contextual | `boolean` | false | If true, will prevent the field from appearing in the dialog box for a widget | [select](field-types/select.md), [checkboxes](field-types/checkboxes.md) | |
```

Comment:
This can be used by any field type.

Reply:
Updated the field to universal and moved it up in the list.

Reply:
Not *every* field type, but any field type whose value is stored as a string. It wouldn't make sense for boolean, checkboxes (because they are stored as arrays), array, object, singleton, area, color, etc. It would be superfluous for `slug`. That leaves `string` and I guess `select`, anything else come to mind Bea?

Reply:
Good point. Reading the code it looked like it was _possible_ on any field type. Advisable is another question, and we should make sure it's locked down to the ones that make sense. I recently ran into a case where another dev had used it to hide the `title` field (since you can't _remove_ it), but we wouldn't encourage its use for simply hiding fields.

I thought that areas/singletons could be marked as contextual since those would be editable in-context. Really they may be the only that would be possible. In reviewing all the projects on my machine, the property is only used when a) the field is an area/singleton or b) on `title` and `slug` to simply hide them when those properties are instead set in `beforeSave`. So the original documentation of it on `select` and `checkbox` was misguided, and we should really only direct people to it for areas and singletons.

The next question is whether to rename or update it in 3.x to take the field type into account.

Reply:
Okay, so to make sure I don't mess this up, we want contextual to only be listed for areas and singletons? And we should remove it from anywhere else that it's listed as an available property (even though it may be *technically* available, we don't want people to use it)?